pxr/usd/usdLux/lightAPI.h

```diff
@@ -435,6 +616,15 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     /// enableColorTemperature is set to true.  When active, the
     /// computed result multiplies against the color attribute.
     /// See UsdLuxBlackbodyTemperatureAsRgb().
+    /// 
+    /// This is always calculated as an RGB color using a D65 white point,
+    /// regardless of the rendering color space, normalized such that the
+    /// default value of 6500 will always result in white, and then should be
+    /// transformed to the rendering color space.
+    /// 
+    /// Spectral renderers should do the same and then uplift the resulting
```

Comment:
"uplift" please imagine my tears and consequent suggestion

Reply:
@meshula @anderslanglands - I was assuming you meant that this sentence could just be removed?

https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-11634bb5438749e3c529c9f44011d464e79bdb1352c0b7fc7c9408bece4415e4L625-L627

Reply:
I worded it this way so as to maintain compatibility between RGB and spectral. Later, when we add illuminants, then we would do this properly. 

The alternative is we just go the whole hog and add in the illuminant spectra now, making one of the options a blackbody, and change the meaning of this to only apply when that illuminant spectrum is selected.

Reply:
Hmm - I restored the sentence* (just so it's more visible for reference in discussion) for now, but it seems we need to reach some more consensus on how to handle this.

*(Sorry if this causes more :eye: :droplet:, @meshula !)

Reply:
This wording is much more specific about avoiding unspecified uplift, so I like the revision.

Reply:
We actually hadn't added any new language to this specific section (`GetColorTemperatureAttr()`) about avoiding unspecified uplift, so I added some (and mirrored the changes to the docs for `GetColorAttr()`).


- [specifically mention desire to avoid unspecified uplight - e12942c](https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/e12942c9740de80af0ed285e4f05527d9de75464)